root/account/Register.js

```diff
@@ -77,14 +77,15 @@ component Register(
 
       <p>
         {exp.l(
-          `Follow our {blog_link|blog} or {twitterx_link|twitter account}!
-           To talk to other users,
+          `Follow our {blog_link|blog} or our {bluesky_link|Bluesky}
+           or {mastodon_link|Mastodon} accounts! To talk to other users,
```

Comment:
Very annoying question: any reason Bluesky is listed here first but second in the footer?

Reply:
Not really - but given they're supposed to be at the same level, maybe it's good to keep them wondering!